templates/youtubedl-material.xml

```diff
@@ -21,6 +21,8 @@ It will also keep a record of already downloaded items. Enjoy! :)</Overview>
   <Config Name="Video Downloads" Target="/app/video/" Default="" Mode="rw" Description="Video Downloads" Type="Path" Display="always" Required="true" Mask="false"/>
   <Config Name="Subscription Downloads" Target="/app/subscriptions/" Default="" Mode="rw" Description="Subscription Downloads" Type="Path" Display="always" Required="true" Mask="false"/>
   <Config Name="WebUI Title" Target="ytdl_title_top" Default="YoutubeDL Material" Mode="" Description="Container Variable: ytdl_title_top" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="User ID" Target="UID" Default="99" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">99</Config>
```

Comment:
Is this `UID` or `PUID`?

Reply:
I just followed what the developer said and it works as expected with those variables:
![Screenshot_20241109_223739_Edge](https://github.com/user-attachments/assets/913b361b-772f-4a64-9be1-4789a15e8319)